packages/remark-codesandbox/src/index.js

```diff
@@ -101,10 +101,19 @@ function codesandbox(options = {}) {
         );
       }
 
+      const overrideEntry =
+        query.get('overrideEntry') !== 'false' ||
+        options.overrideEntry ||
+        true;
+
+      // Remove any options that are only for the plugin and not relevant to CodeSandbox
+      const pluginOnlyQueryParams = ['overrideEntry'];
+      pluginOnlyQueryParams.forEach(query.delete);
```

Comment:
Nitpick: Prefer explicitly call delete with an arrow function inside the `forEach` callback.

To prevent us from passing `index` or other implicit arguments accidentally to `delete` function.

```suggestion
      pluginOnlyQueryParams.forEach(param => {
        query.delete(param);
      });
```